cray_product_catalog/query.py

```diff
@@ -218,17 +244,19 @@ def docker_images(self):
                 for component in self.component_data.get(COMPONENT_DOCKER_KEY) or []]
 
     @property
-    def helm_charts(self):
+    def helm(self):
```

Comment:
Is there a good reason to change this? It would change the expected interface of this class for anything using it. I don't know off the top of my head if anything is using it, but if there's not a good reason to change, then maybe it would be better to leave it alone and avoid potentially breaking things that may depend on it.

Reply:
This was typo changed back to helm_charts